docs/markdown/changelog.raw.md

```diff
@@ -16,6 +16,34 @@ the previous line.
 - [#5094](https://github.com/PowerDNS/pdns/pull/5094): make our zone parser adhere to RFC2308 wrt implicit TTLs and add test
 -->
 
+# PowerDNS Recursor 4.0.6
+Released 6th of July 2017
+
+This release features a fix for the ed25519 verifier. This verifier hashed the message before verifying, resulting in unverifiable signatures. Also on the Elliptic Curve front, support was added for ED448 (DNSSEC algorithm 16) by using libdecaf.
+
+Besides that, this release features massive improvements to our edns-client-subnet handling, and some IXFR fixes. Note that this release changes `use-incoming-edns-subnet` to disabled by default.
+
+## Bug fixes
+- [commit c24288b87](https://github.com/PowerDNS/pdns/commit/c24288b87): rec: Use the incoming ECS for cache lookup if `use-incoming-edns-subnet` is set (Remi Gacogne)
+- [commit b91dc6e92](https://github.com/PowerDNS/pdns/commit/b91dc6e92): when making a netmask from a comboaddress, we neglected to zero the port. This could lead to a proliferation of netmasks. (bert hubert)
+- [commit 261591b6f](https://github.com/PowerDNS/pdns/commit/261591b6f): rec: Don't take the initial ECS source for a scope one if EDNS is off (Remi Gacogne)
+- [commit 66f894b7a](https://github.com/PowerDNS/pdns/commit/66f894b7a): also set d_requestor without Lua: the ECS logic needs it (bert hubert)
+- [commit c2086f265](https://github.com/PowerDNS/pdns/commit/c2086f265): rec: Fix IXFR skipping the additions part of the last sequence (Remi Gacogne)
+- [commit a5c9534d0](https://github.com/PowerDNS/pdns/commit/a5c9534d0): rec: Treat requestor's payload size lower than 512 as equal to 512 (Remi Gacogne)
+- [commit 61b1ea2f4](https://github.com/PowerDNS/pdns/commit/61b1ea2f4): make URI integers 16 bits, fixes [ticket #5443](https://github.com/PowerDNS/pdns/issues/5443) (Peter van Dijk)
+- [commit 27f9da3c2](https://github.com/PowerDNS/pdns/commit/27f9da3c2): unbreak quoting; fixes [ticket #5401](https://github.com/PowerDNS/pdns/issues/5401) (Peter van Dijk)
+
+## Improvements
+- [commit 2325010e6](https://github.com/PowerDNS/pdns/commit/2325010e6): with this, EDNS Client Subnet becomes compatible with the packet cache, using the existing variable answer facility. (bert hubert)
+- [commit 2ec8d8148](https://github.com/PowerDNS/pdns/commit/2ec8d8148): Remove just enough entries from the cache, not one more than asked (Remi Gacogne)
+- [commit 71df15677](https://github.com/PowerDNS/pdns/commit/71df15677): rec: Move expired cache entries to the front so they are expunged (Remi Gacogne)
+- [commit d84834c4c](https://github.com/PowerDNS/pdns/commit/d84834c4c): rec: changed IPv6 addr of b.root-servers.net (Arsen Stasic)
```

Comment:
the 'rec' prefixes are not needed (maybe rewrite/prettify the changelog entries?)